src/CommunityToolkit.Maui.Core/Handlers/Popup/PopupHandler.tizen.cs

```diff
@@ -0,0 +1,103 @@
+﻿using CommunityToolkit.Maui.Core.Views;
+using Tizen.UIExtensions.NUI;
+
+namespace CommunityToolkit.Maui.Core.Handlers;
+
+public partial class PopupHandler : Microsoft.Maui.Handlers.ElementHandler<IPopup, MauiPopup>
+{
+	/// <summary>
+	/// Action that's triggered when the Popup is closed.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	/// <param name="result">The result that should return from this Popup.</param>
+	public static void MapOnClosed(PopupHandler handler, IPopup view, object? result)
+	{
+		var popup = handler.PlatformView;
+
+		if (popup.IsOpen)
+		{
+			popup.Close();
+		}
+
+		handler.DisconnectHandler(popup);
+	}
+
+	/// <summary>
+	/// Action that's triggered when the Popup is Opened.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	/// <param name="result">We don't need to provide the result parameter here.</param>
+	public static void MapOnOpened(PopupHandler handler, IPopup view, object? result)
+	{
+		handler.PlatformView.ShowPopup();
+	}
+
+	/// <summary>
+	/// Action that's triggered when the Popup is dismissed by tapping outside of the Popup.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	/// <param name="result">The result that should return from this Popup.</param>
+	public static void MapOnDismissedByTappingOutsideOfPopup(PopupHandler handler, IPopup view, object? result)
+	{
+		if (view.CanBeDismissedByTappingOutsideOfPopup)
+		{
+			view.OnDismissedByTappingOutsideOfPopup();
+		}
+	}
+
+	/// <summary>
+	/// Action that's triggered when the Popup <see cref="IPopup.Anchor"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	public static void MapAnchor(PopupHandler handler, IPopup view)
+	{
+		// On Tizen, Anchor only update when popup is opened
+	}
+
+	/// <summary>
+	/// Action that's triggered when the Popup <see cref="IPopup.CanBeDismissedByTappingOutsideOfPopup"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="PopupHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IPopup"/>.</param>
+	public static void MapCanBeDismissedByTappingOutsideOfPopup(PopupHandler handler, IPopup view)
```

Comment:
Since `Tizen` doesn't implement those methods, can you remove them from the mapper? Following [this pattern](https://github.com/dotnet/maui/blob/main/src/Core/src/Handlers/Label/LabelHandler.cs#L20-L23)

Reply:
IMO, Empty method it better than conditional build flags. and [this pattern](https://github.com/dotnet/maui/blob/main/src/Core/src/Handlers/Label/LabelHandler.cs#L20-L23) is not same to current case. 
because this pattern is used to add mapper for some specific platform. this this case to remove only some specific platform

for example [MissingMapper](https://github.com/dotnet/maui/blob/a2947dfcce1e2629fe1946b120070f3c38ce0bb0/src/Core/src/Handlers/Label/LabelHandler.Tizen.cs#L61) is commonly used on MAUI